conwaylib/GameState.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Drawing;
+
+namespace ConwayLib
+{
+    public enum DensityOption {Dense, Sparse}
+    public class GameState
+    {
+        [JsonPropertyName("Format")]
+        public string FormatText {get; set;}
+        [JsonIgnore]
+        public DensityOption Format {
+            get
+            {
+                return Enum.Parse<DensityOption>(FormatText);
```

Comment:
This property could be a single line

Reply:
The new version is one line but basically has the same content. My point was that you could just have used `=>`

However, it's not worth further to-and-fro, so I will approve on this occasion.